### DISCUSSION TITLE
请问该项目支持 极路由X 设备吗？

### DISCUSSION THREAD

**Discussion by JIUJIUs** [istoreos/istoreos#817] [General]
请问该项目支持 极路由X 设备吗？
设备型号：c562a （极路由X）
cpu信息：ipq40xx

**Comment by rootphantomer** [comment]
https://fw.koolcenter.com/iStoreOS/

就支持这些
![image](https://github.com/istoreos/istoreos/assets/6481352/3ce369da-ea2f-49e7-8ad9-d064aba70be8)

**Reply by JIUJIUs** [reply]
谢回复。希望可以支持上~

**Reply by rootphantomer** [reply]
他们只支持arm软路由和X86架构的，目前还没适配过硬路由。

**Reply by jjm2473** [reply]
硬路由的存储空间以及CPU性能都比较差，官方不打算支持。如果有兴趣的话，可以自己编译，之前GL-iNet就有打算移植iStoreOS的部分功能，我们也提供了arm架构的包，不知道他们后来有没有继续搞

**Reply by JIUJIUs** [reply]
好的，谢谢大佬回复！

**Comment by jjm2473** [comment]
lean可能会有些高通机型的支持，不过硬路由可能会缺少一些功能，例如升级固件保留插件，沙箱模式等。
